pontoon/base/views.py

```diff
@@ -281,6 +281,22 @@ def entities(request):
     return _get_paginated_entities(locale, preferred_source_locale, project, form, entities)
 
 
+def _serialize_translation_values(query):
+    return [
+        {
+            'locale': {
+                'code': translation.locale.code,
+                'direction': translation.locale.direction,
+                'name': translation.locale.name,
+                'pk': translation.locale.pk,
+                'script': translation.locale.script,
+            },
```

Comment:
I just realized we have a `serialize()` method on the `Locale` class. If you add `pk` to it, you could just to this:
```python
'locale': translation.locale.serialize(),
```

And for the source string:
```python
'locale': Locale.objects.get(code='en-US').serialize(),
```